src/node_kernels/TKEKENodeKernel.C

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#include "node_kernels/TKEKENodeKernel.h"
+#include "Realm.h"
+#include "SolutionOptions.h"
+#include "SimdInterface.h"
+#include "utils/StkHelpers.h"
+
+#include "stk_mesh/base/MetaData.hpp"
+#include "stk_mesh/base/Types.hpp"
+
+namespace sierra {
+namespace nalu {
+
+TKEKENodeKernel::TKEKENodeKernel(const stk::mesh::MetaData& meta)
+  : NGPNodeKernel<TKEKENodeKernel>(),
+    tkeID_(get_field_ordinal(meta, "turbulent_ke")),
+    tdrID_(get_field_ordinal(meta, "total_dissipation_rate")),
+    densityID_(get_field_ordinal(meta, "density")),
+    tviscID_(get_field_ordinal(meta, "turbulent_viscosity")),
+    viscID_(get_field_ordinal(meta, "viscosity")),
+    dudxID_(get_field_ordinal(meta, "dudx")),
+    wallDistID_(get_field_ordinal(meta, "minimum_distance_to_wall")),
+    dualNodalVolumeID_(get_field_ordinal(meta, "dual_nodal_volume")),
+    nDim_(meta.spatial_dimension())
+{
+}
+
+void
+TKEKENodeKernel::setup(Realm& realm)
+{
+  const auto& fieldMgr = realm.ngp_field_manager();
+
+  tke_ = fieldMgr.get_field<double>(tkeID_);
+  tdr_ = fieldMgr.get_field<double>(tdrID_);
+  density_ = fieldMgr.get_field<double>(densityID_);
+  visc_ = fieldMgr.get_field<double>(viscID_);
+  tvisc_ = fieldMgr.get_field<double>(tviscID_);
+  dudx_ = fieldMgr.get_field<double>(dudxID_);
+  wallDist_ = fieldMgr.get_field<double>(wallDistID_);
+  dualNodalVolume_ = fieldMgr.get_field<double>(dualNodalVolumeID_);
+
+  const std::string dofName = "turbulent_ke";
+  relaxFac_ = realm.solutionOptions_->get_relaxation_factor(dofName);
```

Comment:
should we be using this?

Reply:
we don't use it in SST. This comes from a copy/paste thing. Removed